manifests/charts/gateways/istio-ingress/values.yaml

```diff
@@ -125,7 +128,10 @@ gateways:
     podAntiAffinityTermLabelSelector: []
 
     # whether to run the gateway in a privileged container
-    runAsRoot: false
+    # At this time, because of several crash related issues, the Istio community strongly recommends
+    # running the gateway as root. See:
+    # https://github.com/istio/istio/issues/27566 and https://github.com/istio/istio/issues/26690
+    runAsRoot: true
```

Comment:
We will need to fix the operator bug before doing this I think. i am worried about people getting in inconsistent states

Reply:
Sounds like a fair dependency to me.

Reply:
I've added a Depends-On to the description.

Cheers,
-steve